### PR TITLE
add tiup link doc

### PR DESCRIPTION
**Pull request by kaaaaaaang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by srstack — APPROVED** [review]

File: tiup/tiup-component-management.md [hunk 1]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
 ```shell
 tiup uninstall --all
 ```
+
+### 链接组件
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:245]: 请把 link/unlink 添加到这篇文档开头第一段的列表里~

File: tiup/tiup-component-management.md [hunk 2]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+
+TiUP v1.13.0 添加了实验性的 `link` 和 `unlink` 命令，用于将组件的二进制符号链接到可执行文件目录（$TIUP_HOME/bin/）和删除链接。这个功能可以让用户不必在每次都经过 tiup 来调用组件，同时保留了多版本切换的能力。这种方式缺少了自动检查更新和设置某些环境变量的过程，一些组件例如 ctl 可能无法使用，请只在必要的场景来使用它。
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:247]:
```suggestion
TiUP v1.13.0 添加了实验性的 `link` 和 `unlink` 命令，用于将组件的二进制符号链接到可执行文件目录 (`$TIUP_HOME/bin/`) 和删除链接。这个功能可以让用户不必在每次都经过 TiUP 来调用组件，同时保留了多版本切换的能力。然而，这种方式缺少了自动检查更新和设置某些环境变量的过程，一些组件（例如 ctl）可能无法使用，请只在必要的场景来使用它。
```

File: tiup/tiup-component-management.md [hunk 3]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+
+示例一： 安装并链接 cluster 组件的最新版本
+
+{{< copyable "shell-regular" >}}
+
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:252]:
```suggestion
```

File: tiup/tiup-component-management.md [hunk 4]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
 ```shell
 tiup uninstall --all
 ```
+
+### 链接组件
+
+TiUP v1.13.0 添加了实验性的 `link` 和 `unlink` 命令，用于将组件的二进制符号链接到可执行文件目录（$TIUP_HOME/bin/）和删除链接。这个功能可以让用户不必在每次都经过 tiup 来调用组件，同时保留了多版本切换的能力。这种方式缺少了自动检查更新和设置某些环境变量的过程，一些组件例如 ctl 可能无法使用，请只在必要的场景来使用它。
+
+示例一： 安装并链接 cluster 组件的最新版本
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:249]:
```suggestion
示例一：安装并链接 cluster 组件的最新版本
```

File: tiup/tiup-component-management.md [hunk 5]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+
+```shell
+tiup install cluster
+tiup link cluster
+```
+
+示例二： 切换 cluster 组件到 v1.13.0 版本
+
+{{< copyable "shell-regular" >}}
+
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:261]:
```suggestion
```

File: tiup/tiup-component-management.md [hunk 6]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+示例一： 安装并链接 cluster 组件的最新版本
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup install cluster
+tiup link cluster
+```
+
+示例二： 切换 cluster 组件到 v1.13.0 版本
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:258]:
```suggestion
示例二：切换 cluster 组件到 v1.13.0 版本
```

File: tiup/tiup-component-management.md [hunk 7]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+tiup link cluster：v1.13.0
+```
+
+`tiup link cluster` 命令会输出如下的内容
+
+```shell
+package cluster provides these executables: tiup-cluster
+```
+
+代表 cluster 组件的二进制文件名为 tiup-cluster,link 命令完成后可以直接在命令行敲击 tiup-cluster 来使用 cluster 组件。
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:272]:
```suggestion
代表 cluster 组件的二进制文件名为 tiup-cluster。link 命令完成后，你可以直接在命令行输入 tiup-cluster 来使用 cluster 组件。
```

File: tiup/tiup-component-management.md [hunk 8]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+
+示例三： 取消 cluster 组件的链接
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup unlink cluster
+```
+
+示例四： 对 tiup 自身进行版本管理
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:282]:
```suggestion
示例四：对 TiUP 自身进行版本管理
```

File: tiup/tiup-component-management.md [hunk 9]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+
+在 v1.13.0 之前的版本，tiup 自身被安装到 ~/.tiup/bin/ 下，不能多版本共存。自 v1.13.0 开始，可以将它和所有其他组件一样进行安装、链接。
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:284]:
```suggestion
在 v1.13.0 之前的版本，TiUP 自身被安装到 `~/.tiup/bin/` 下，不能多版本共存。自 v1.13.0 开始，可以将 TiUP 和所有其他组件一样进行安装、链接。
```

File: tiup/tiup-component-management.md [hunk 10]
```diff
@@ -241,3 +241,51 @@ tiup uninstall tikv --all
...
+
+{{< copyable "shell-regular" >}}
+
```
Comment (ran-huang) [inline tiup/tiup-component-management.md:287]:
```suggestion
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**